apps/backend/src/datasources/postgres/CallDataSource.test.ts

```diff
@@ -22,42 +25,40 @@ async function getProposalsInCall(callId: number): Promise<Proposal[]> {
   return proposals.map((p) => createProposalObject(p));
 }
 
-async function createCall(format?: string): Promise<Call> {
-  const call = await database('call')
-    .modify((query) => {
-      if (format) {
-        query.insert({
-          call_id: -999,
-          call_short_code: '[IT] call',
-          start_call: new Date(),
-          end_call: new Date(),
-          start_review: new Date(),
-          end_review: new Date(),
-          start_notify: new Date(),
-          end_notify: new Date(),
-          cycle_comment: '',
-          survey_comment: '',
-          start_cycle: new Date(),
-          end_cycle: new Date(),
-          reference_number_format: format,
-        });
-      } else {
-        query.insert({
-          call_id: -999,
-          call_short_code: '[IT] call',
-          start_call: new Date(),
-          end_call: new Date(),
-          start_review: new Date(),
-          end_review: new Date(),
-          start_notify: new Date(),
-          end_notify: new Date(),
-          cycle_comment: '',
-          survey_comment: '',
-          start_cycle: new Date(),
-          end_cycle: new Date(),
-        });
-      }
-    })
+async function createCall(args: {
+  format?: string;
+  callEnded?: boolean;
+  callEndedInternal?: boolean;
+  endCall?: Date;
+  endCallInternal?: Date;
+}): Promise<Call> {
```

Comment:
A suggestion, maybe we can use default value if no argument is passed? Up to you.
```suggestion
async function createCall(args: {
  format?: string;
  callEnded?: boolean;
  callEndedInternal?: boolean;
  endCall?: Date;
  endCallInternal?: Date;
} = {}): Promise<Call> {
```

Reply:
I didn't know about this, thanks! It is nicer